data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1791 @@
+			`	Another one bites into something that looks like a goat's leg, albeit much bonier, before saying, "Prepared for concealing our presence, we and the crew are. Worry about that, you need not."`
+			`	"Extra fuel, the ship has. Enough to reach the planet, it is," the last of them tells you. You stay at the restaurant for a surprisingly short while, as the Arachi devour the full-course meals in only a few minutes. You leave together, accompanying them to the hangars, where you see their Spindle has landed next to the <ship>. They board it, saying they will follow your ship once you depart, and that you may take off at your earliest convenience.`
+				accept
+	npc accompany save
+		personality escort timid
+		government "Coalition"
+		ship "Arach Spindle (Jump Drive)" "Toopy Mogrup"
+	on visit
+		dialog `You've reached <planet>, but left House Bebliss' ship behind! Best depart and wait for them to get here.`
+	on complete
+		event "bebliss research fw" 80 100
+		payment 930000
+		conversation
+			`As both your ships come in for a landing, you contact Freya and tell her the Arachi are with you this time. She tells you to land near the Free Worlds' warehouses, where you last met her, saying she'll be waiting there to greet them.`
+			`	When you arrive, you wait outside your ship while House Bebliss' ship keeps its hatch closed. It seems they intend to remain in there for the duration of their stay, most likely in fear that their appearance would cause a stir amongst other humans.`
+			`	Freya arrives and the hatch opens for both of you to go inside once no curious bystanders are nearby. Freya sees the members of House Bebliss and confidently walks up to them. "Hello, my name is Freya Winters. I've heard you're interested in the Pug artifacts we found on this world." For someone speaking with giant spiders for the first time, she handles the initial meeting surprisingly well.`
+			`	"Indeed. The trade of our great House Bebliss for thousands of years, the study of hyperlanes has been," one of the Arachi says. "When Captain <last> told us of your achievements, know that need to come look into your work ourselves, we did."`
+			`	She quickly gets used to the translation box's odd speech quirk, as she and the members of House Bebliss discuss how to go about helping each other in researching the artifacts here. One of the crewmembers approaches you as they talk, and hands you <payment>, "as thanks for safely escorting us here," they whisper.`
+			`	Freya's talk with them goes on for almost an hour; they eventually settle on having some of the artifacts moved to their ship's cargo hold, so that some research can happen there without the risk of being seen. "Hopefully people buy into this being a new niche ship design that Southbound Shipyards asked me to check out," Freya says. House Bebliss' representatives thank her for the reception, and you follow her back outside.`
```

Comment:
I've just removed that silly bit.

Since Deneb is not within the jurisdicition of any of the three groups, and they all have teams there, I imagine there's the occasional "this is classified stuff, we're not letting you in on that" from one to the other two, so this'd be a case of the FW doing that.

I was going to add that as the replacement explanation, but felt it'd be drawn out. I'll add it in if you feel it's necessary.